scripts/scan/clang-scan-build-xtensa.sh

```diff
@@ -70,7 +70,7 @@ rm -fr $BUILD_DIR
 mkdir $BUILD_DIR
 cd $BUILD_DIR
 
-scan-build $VERBOSE_SCAN_BUILD cmake \
+scan-build-11 $VERBOSE_SCAN_BUILD cmake \
```

Comment:
I think this maybe better as a shell variable that can by default be "11", but we should have command line option to change it to say a different version.